test/modules/@company/bundle/ui/src/company.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
`return 'BUNDLE_OVERRIDE_COMPANY'` doesn't work?

Reply:
Nope it didn't, I don't know why it was stripped from the final file, I gave up